packages/core/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# 0.1.0+1
+
+- feat: initial commit 🎉
```

Comment:
Since you mentioned that you are curious about the whole versioning/release functionality I think it would be great if you could look into `melos` versioning

A first entry point for understanding how it goes could be: https://melos.invertase.dev/commands/version

Reply:
I will look into this and work on it in a separate PR.